### PR TITLE
Update `sanctuary-def` dependency to v0.14.0

### PR DESCRIPTION
**Pull request by brusherru**
There is no issue.

### CODE REVIEW

File: packages/xod-client/src/editor/selectors.js [hunk 1]
```diff
@@ -52,6 +52,29 @@ export const getCurrentPatchOffset = createSelector(
   R.propOr(DEFAULT_PANNING_OFFSET, 'offset')
 );
 
+// :: State -> EditorTabs
+export const getPreparedTabs = createSelector(
```
Comment (nkrkv) [inline packages/xod-client/src/editor/selectors.js]: How is this related to the bump?

Please, review a PR by yourself as a first step before posting the PR

Comment (brusherru) [reply packages/xod-client/src/editor/selectors.js]: Just a little refactoring and optimization.
Some other functions are fixed to work properly with turned on checkTypes.

File: packages/xod-client-electron/src/view/containers/App.jsx [hunk 2]
```diff
@@ -233,12 +234,18 @@ class App extends client.App {
               client.composeMessage(error.message)
             ),
             (nodeIdsMap) => {
-              this.props.actions.startDebuggerSession(
-                createSystemMessage('Debug session started'),
-                nodeIdsMap,
+              Maybe.maybe(
+                null,
+                (currentPatchPath) => {
```
Comment (nkrkv) [inline packages/xod-client-electron/src/view/containers/App.jsx]: Side-effect maybe. I think `isNothing` + `explode` will be more appropriate here.

File: packages/xod-client/src/editor/messages.js [hunk 3]
```diff
@@ -16,3 +16,8 @@ export const clipboardMissingPatchPasteError = missingPatches => composeMessage(
 
 export const LIB_SUGGESTER_TYPE_TO_BEGIN = 'Type owner/libname to find a library';
 export const LIB_SUGGESTER_NOTHING_FOUND = 'No library found';
+
+export const OPEN_PATCH_TO_TRANSPILE_FIRST = composeMessage(
+  'Canʼt transpile code',
+  'Open Patch to transpile first'
+);
```
[the remark this answers is not in the document]
Comment (nkrkv) [reply packages/xod-client/src/editor/messages.js]: And… a simple identifier like `NO_PATCH` will be better IMO

File: packages/xod-client/src/editor/containers/Editor.jsx [hunk 4]
```diff
@@ -108,54 +113,69 @@ class Editor extends React.Component {
...
+    return foldMaybe(
+      <NoPatch />,
+      (tab) => {
+        // Do not render <Patch /> component if opened tab
+        // is in EditingCppImplementation mode.
+        if (tab.isEditingCppImplementation) return null;
+
+        // If we reached here, we're sure that tab contains
+        // all the necessary data and just explode them up
+        // to pass it into <Patch />
```
Comment (evgenykochetkov) [inline packages/xod-client/src/editor/containers/Editor.jsx]: Nice comments 👍🏻
